yellowbrick/features/base.py

```diff
@@ -218,7 +218,7 @@ def fit(self, X, y=None, **kwargs):
         # Store the classes for the legend if they're None.
         if self.classes_ is None:
             # TODO: Is this the most efficient method?
-            self.classes_ = [str(label) for label in set(y)]
+            self.classes_ = [str(label) for label in np.unique(y)]
```

Comment:
I've been wanting to do this a while, but I tried it once and it broke our tests -- glad to see it's working with the tests now; guess we've made a bit of progress. I think the issue was that np.unique` orders the return value alphabetically, where as `set` does not guarantee any particular order.